pymdownx/blocks/block.py

```diff
@@ -30,39 +35,39 @@ def _type_multi(value, types=None):
     raise ValueError(f"Type '{type(value)}' did not match any of the provided types")
 
 
-def type_multi(*args):
+def type_multi(*args: Callable[[Any], _T]) -> Callable[[Any], _T]:
```

Comment:
Isn't `*args` an interable?

Reply:
The way they set up the type system is that for `*varargs` the type is of the element of the n-tuple

Reply:
Yeah, you're right. Now I remember I've done this as well in the past.